data-sources/apache-flink/config.yml

```diff
@@ -1,7 +1,7 @@
 id: apache-flink
 displayName: Apache Flink monitoring
 description: |
-  New Relic’s instant observability quickstart with Apache Flink monitoring helps to detect client, jobmanager and taskmanager logs and parses them into a JSON payload.
+  With our Apache Flink dashboard, you can easily track your logs, keep an eye on your instrumentation sources, and get an overview of uptime and downtime for all your applications' instances. Built with our infrastructure agent and our Prometheus OpenMetrics integration, Apache Flink dashboards take advantage of OpenMetrics endpoint scraping, so you can view all your most important data in one place.
```

Comment:
This description is too long. It should be 240 characters max

Reply:
Thank you @brammerl , I have updated it.